website/server/models/user/methods.js

```diff
@@ -299,3 +300,24 @@ schema.methods.canGetGems = async function canObtainGems () {
     return !g.isSubscribed() || g.leader === user._id || g.leaderOnly.getGems !== true;
   });
 };
+
+const officialPinnedItems = common.content.officialPinnedItems;
+
+schema.methods.togglePinnedItem = function togglePinnedItem (key) {
```

Comment:
like this could go in a common ops file so that it can be used both on server and client